packages/backend/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@backstage/backend",
```

Comment:
Maybe just call this `backend` or smth for now while we experiment? Just to avoid confusion that this would be a published package.

Reply:
I thought of that. I thought that this could be our pattern for now and then be changed to @internal.

But you're right and it stood probably be example-backend or so